connector/connect/server/src/main/scala/org/apache/spark/sql/connect/config/Connect.scala

```diff
@@ -47,6 +47,13 @@ object Connect {
       .bytesConf(ByteUnit.MiB)
       .createWithDefaultString("4m")
 
+  val CONNECT_GRPC_MAX_INBOUND_MESSAGE_SIZE =
+    ConfigBuilder("spark.connect.grpc.maxInboundMessageSize")
```

Comment:
If this is a user-facing conf, should probably document at https://github.com/apache/spark/blob/master/docs/configuration.md#spark-connect

Reply:
+1 for documenting.

Reply:
+1

Reply:
Done